velox/type/DecimalUtil.h

```diff
@@ -250,8 +250,24 @@ class DecimalUtil {
     return (unsignedSum >> 127);
   }
 
+  /// Adds two int128_t inputs, calculates the sum, and returns the overflow.
+  /// It can be used to track the number of overflow when adding a batch of
```

Comment:
Why is it useful to track number of overflows? Are we saying that one positive overflow cancels one negative overflow? Why is that the case?

Reply:
Yes. The logic is that one positive overflow cancels one negative overflow. Overflows typically only affect the higher bits, while the lower bits of the result remain correct. If the number of positive overflows and negative overflows can cancel each other, then the final result will be correct, or we can adjust it to a correct result.

Reply:
Got it. Thank you for clarifying. It would be nice to add this to the comment so other readers can understand what's happening more quickly.